src/WeatherRequest.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react';
+
+const currentWeatherURI = 'https://api.openweathermap.org/data/2.5/weather';
+const hourlyWeatherURI =
+  'https://pro.openweathermap.org/data/2.5/forecast/hourly';
+const dailyWeatherURI =
+  'https://api.openweathermap.org/data/2.5/forecast/daily';
+
+const requestLonLat = async function requestWeatherByLonAndLat(URIparams, URI) {
```

Comment:
Do you have a preference when it comes to function declarations? I usually use the array syntax `const requestLonLat = async (URIparams,URI) => {...}` because it seems shorter but they both follow the airbnb style guide. Just want to make sure we're consistent.

Reply:
I tend to like the named function expressions just because the descriptive name helps clarify exactly what's going on, but I also tend to err towards verbosity lol. I also don't think if you'd like to continue using arrow functions following the style guide that it'd necessarily result in too much inconsistency in our codebase.